include/geometry/triangle.h

```diff
@@ -0,0 +1,22 @@
+#ifndef GEOMETRY_TRIANGLE_H
+#define GEOMETRY_TRIANGLE_H
+
+#include <geometry/point.h>
+#include <geometry/vetor.h>
+
+struct Triangle {
+    Point vertices[3]; 
+
+    Triangle();
+    Triangle(const Point& v1, const Point& v2, const Point& v3);
+
+    double area() const;  // area of the triangle
+    Point centroid() const;  // centroid of the triangle
+    Vetor normal() const;  // normal vector of the triangle
+    bool contains(const Point& p) const;  // check if the triangle contains a point
+
+    bool operator==(const Triangle& other) const;
+    bool operator!=(const Triangle& other) const;
+};
+
+#endif
```

Comment:
Para mim faz sentido dropar isso daqui

```suggestion
```

Reply:
@mateuseap eu gostei disso. Se a gente fosse de fato utilizar, seria uma solução para os construtores pesados. A gente em alguma hora vai precisar pegar o baricentro da superfície (acho que no projeto de bézier), e ter uma estrutura dessa facilitaria muito. 

os métodos `contains` e `centroid` por exemplo são utéis e a gente vai precisar usar no `intersect`.

Sugestão de agora: criar um método `getTriangle` no `TriangularMesh` que recebe um index (índice do triângulo) e retorna essa estrutura.

abs.: pode tirar os comentários, acho que os nomes dos métodos tão de boa pra entender.

Reply:
Beleza, vou fechar essas threads e sinalizar p remover os comentários